### DISCUSSION TITLE
Welcome to Eliza Chat!

### DISCUSSION THREAD

**Discussion by awidearray** [elizaOS/eliza#392] [Announcements]
**Welcome to Eliza Chat!**
==========================

We're excited to have you join the Eliza community! This chat is a space for developers, contributors, and users to discuss the Eliza project, ask questions, and share knowledge.

**What is Eliza?**
-----------------

Eliza is an open-source project that aims to be the premiere open-source AI chatbot/agent framework. Eliza aims to democratize AI agent development by providing a robust, extensible framework that empowers developers to create sophisticated AI applications without getting bogged down in infrastructure. We believe that AI agents should be:

- **Accessible**: Easy to set up and use, regardless of expertise level
- **Adaptable**: Flexible enough to work with any AI model or platform
- **Powerful**: Capable of handling complex tasks and interactions
- **Community-Driven**: Built by and for developers worldwide

**How to Get Started**
--------------------

1. **Read the docs**: Check out our [documentation](https://ai16z.github.io/eliza/) to learn more about Eliza and how to use it.
2. **Join the conversation**: Introduce yourself, ask questions, and share your thoughts on the project.
3. **Contribute**: If you're interested in contributing to Eliza, please see our [contributing guide](https://github.com/ai16z/eliza/blob/main/CONTRIBUTING.md).

**Community Guidelines**
------------------------

We strive to create a welcoming and inclusive community. Please see our [code of conduct](https://github.com/ai16z/eliza/blob/main/CONTRIBUTING.md#code-of-conduct) for more information.

**Channels**
------------

* **General**: General discussion about Eliza and its ecosystem.
* **Development**: Discussion about Eliza's development, including features, bugs, and pull requests.
* **Support**: Get help with using Eliza or report issues.

**Thanks for Joining!**
------------------------

We're glad you're here! If you have any questions or need help, feel free to ask.

---

**Eliza Team**

**Comment by yuji4091** [comment]
yo

**Comment by frontboat** [comment]
anybody know how to get my agent to stop posting soooo much on twitter lmao

**Reply by sentilabs01** [reply]
adjust the inteval times in the .env file

**Reply by ReMeDy-TV** [reply]
lol I know this is an old comment, but just wanted to say you made me excited wanting to try this now. Would be funny having an unhinged Twitter bot.

**Reply by frontboat** [reply]
lol it was popping off. daydreams agents too

**Comment by ZeKroR** [comment]
A hoe

**Comment by sentilabs01** [comment]
Getting this build error... anyone else>?
![Screenshot 2025-02-08 181334](https://github.com/user-attachments/assets/50f786a6-54a9-4678-8351-5b5685d4e495)

 Tasks:    96 successful, 141 total
Cached:    96 cached, 141 total
  Time:    9.171s 
Failed:    client#build

 ERROR  run failed: command  exited (1)
 ELIFECYCLE  Command failed with exit code 1.

**Comment by setmisi** [comment]
there is a discord channel?

**Comment by Voi-lu** [comment]
From https://www.elizaos.ai/framework  ---> Community made ---> https://eliza.gg/ (is not working any more). Can you please fix that?

**Comment by MandesHubble** [comment]
Does anyone know how to Auto tweet based on data? Similar to knowledge but I can post tweet about the data.md in knowledge that do not need any conversation.

**Comment by RobertYoung2022** [comment]
The documentation link is giving a 404 error.

**Reply by mrasmuson** [reply]
Go to the Documentation link in the readme file.

**Comment by lloydy927** [comment]
Hi, Eliza support team —

I’m logged in via X as @Lloydy927 and I'm having issues with my in-app wallet. The wallet briefly displays with correct balances, but then disappears from the interface.

On-chain, the wallet is active and fully funded:

Address: 2YmSwA4vDKRVGdMBBQfux7XUMHzzkTGtQASrhe3mBe2F

SOL Balance: 0.01

ELIZA Token: 1K

Verified on Solscan

Can you please help me re-sync or reinitialize my wallet so I can start chatting? Thank you!

**Comment by cshawcross** [comment]
Sure

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by talasusameer** [reply]
include a burn mechanism is useful and while itbring attention and hipe  the project towards future growth

**Comment by landervanpassel-design** [comment]
I'm intrigued by Eliza's mission to democratize AI agent development through accessibility and adaptability. This aligns closely with the principles of Protected Desire Equilibrium (PDE), where autonomous agents form mutual defense treaties to ensure stability and cooperation as an Evolutionarily Stable Strategy. Eliza's framework could serve as a practical platform for implementing PDE, enabling agents to negotiate and uphold such treaties in diverse environments. If you're exploring agent interactions or conflict resolution mechanisms, PDE might offer a theoretical foundation to enhance Eliza's community-driven goals. Let's discuss how these concepts could intersect in your framework.
